src/reference/asciidoc/aggregator.adoc

```diff
@@ -846,8 +846,7 @@ If the flag is set to true, then when the expiry callback is invoked, any unmark
 
 [IMPORTANT]
 =====
-When using a `MessageGroupStoreReaper`, it is generally recommended to use a separate `MessageStore` for each correlating endpoint.
-Otherwise, unexpected results may occur because one endpoint may remove another endpoint's groups.
+When using a `MessageGroupStoreReaper`, with a common `MessageStore` for multiple correlating endpoints, ensure proper `CorrelationStrategy` is used when same `MessageStore` is used for multiple aggregators to ensure unique correlationkey is being assigned to the messages.
```

Comment:
This sentence is a bit awkward.
Plus would be better to place here an answer for the question "Why?".
That  might not be so clear for end-users what is the reason to worry about this restriction.